<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FDIRControl</class>
 <widget class="QWidget" name="FDIRControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>271</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="statusLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="fdir1IMULabel">
      <property name="text">
       <string>FDIR Level 1 IMU Failures:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QSpinBox" name="fdir1IMUSpinBox">
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="maximum">
       <number>1000000</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="Led" name="imuPosErrorLed" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="Led" name="imuStatusLed" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="Led" name="imuFailureLed" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="Led" name="iinConfLed" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="Led" name="primGPSNotNavLed" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="Led" name="primGPSNotAvLed" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="Led" name="primDataGapLed" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="Led" name="primGPSObsLed" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="Led" name="primGPSTimeRecLed" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="Led" name="primGPSPPSLed" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="Led" name="primGPSSNRLed" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="Led" name="baseGPSDataGapLed" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="Led" name="baseGPSParityLed" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="Led" name="baseGPSMsgRejectLed" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="Led" name="secGPSDataGapLed" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="Led" name="secGPSSNRLed" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="Led" name="secGPSObsDataGapLed" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="Led" name="secGPSExcessLockLed" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="Led" name="gamsAmbLed" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>460</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="Led" name="primGPSMissingLed" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="Led" name="primGPSExcLockLed" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="Led" name="primEphemLed" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>171</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>IMU-POS checksum error</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>161</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Successive IMU failures</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>IIN configuration mismatch failure</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>IMU status set by IMU</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>301</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Primary GPS time recovery data not received</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>320</y>
     <width>201</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Base GPS data gap</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>181</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Primary GPS PPS time gap</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>271</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Primary GPS not available for alignment</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>241</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Primary GPS not in Navigation mode</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>201</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Primary data gap</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>460</y>
     <width>241</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>GAMS ambiguity resolution failed</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>420</y>
     <width>181</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Secondary GPS SNR failure</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>440</y>
     <width>281</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Secondary GPS excessive lock-time resets</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>400</y>
     <width>241</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Secondary GPS observable data gap</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>340</y>
     <width>161</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Base GPS parity error</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>380</y>
     <width>161</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Secondary GPS data gap</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>360</y>
     <width>241</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Base GPS message rejected</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>300</y>
     <width>171</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Primary GPS SNR failure</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>280</y>
     <width>221</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Primary GPS missing ephemeris</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>261</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Primary GPS excessive lock-time resets</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>261</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Primary ephemeris data gap</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>271</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Primary GPS observable data gap</string>
   </property>
  </widget>
  <widget class="Led" name="primGPSRejectedLed" native="true">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>200</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="Led" name="primGPSOmniXPLed" native="true">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>440</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>320</y>
     <width>211</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Z gyro failure</string>
   </property>
  </widget>
  <widget class="Led" name="excessPOSErrorLed" native="true">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>380</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_24">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>160</y>
     <width>161</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>GAMS heading rejected</string>
   </property>
  </widget>
  <widget class="Led" name="dmiRejectedLed" native="true">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>180</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_25">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>40</y>
     <width>221</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Inertial speed exceeds maximum</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_26">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>360</y>
     <width>261</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Excessive primary GPS lever arm error</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_27">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>380</y>
     <width>231</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Excessive POS position error RMS</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_28">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>460</y>
     <width>241</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Primary GPS in Omnistar VBS mode</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_29">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>180</y>
     <width>191</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>DMI data rejected</string>
   </property>
  </widget>
  <widget class="Led" name="inertialMaxLed" native="true">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>40</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="Led" name="excessPrimGPSClockLed" native="true">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>400</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_30">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>260</y>
     <width>161</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Z accelerometer failure</string>
   </property>
  </widget>
  <widget class="Led" name="gamsRejectedLed" native="true">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>160</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_31">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>300</y>
     <width>261</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Y gyro failure</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_32">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>280</y>
     <width>161</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>X gyro failure</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_33">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>240</y>
     <width>161</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Y accelerometer failure</string>
   </property>
  </widget>
  <widget class="Led" name="excGAMSOffsetLed" native="true">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>340</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_34">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>400</y>
     <width>231</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Excessive primary GPS clock drift</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_35">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>420</y>
     <width>231</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Primary GPS in Omnistar HP mode</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_36">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>140</y>
     <width>201</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Primary GPS velocity rejected</string>
   </property>
  </widget>
  <widget class="Led" name="dmiSpeedMaxLed" native="true">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>100</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="Led" name="yAccFailureLed" native="true">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>240</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="Led" name="zGyroFailureLed" native="true">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>320</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_37">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>220</y>
     <width>161</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>X accelerometer failure</string>
   </property>
  </widget>
  <widget class="Led" name="primGPSPosRejectedLed" native="true">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>120</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_38">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>120</y>
     <width>201</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Primary GPS position rejected</string>
   </property>
  </widget>
  <widget class="Led" name="xGyroFailureLed" native="true">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>280</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_39">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>200</y>
     <width>241</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Primary GPS observables rejected</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_40">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>80</y>
     <width>301</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Primary GPS position error exceeds maximum</string>
   </property>
  </widget>
  <widget class="Led" name="excessPrimGPSErrorLed" native="true">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>360</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="Led" name="primGPSOmniVBSLed" native="true">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>460</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="Led" name="primGPSVelMaxLed" native="true">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>60</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_41">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>440</y>
     <width>231</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Primary GPS in Omnistar XP mode</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_42">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>60</y>
     <width>271</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Primary GPS velocity exceeds maximum</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_43">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>100</y>
     <width>211</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>DMI speed exceeds maximum</string>
   </property>
  </widget>
  <widget class="Led" name="yGyroFailureLed" native="true">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>300</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_44">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>340</y>
     <width>221</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Excessive GAMS heading offset</string>
   </property>
  </widget>
  <widget class="Led" name="zAccFailureLed" native="true">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>260</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="Led" name="xAccFailureLed" native="true">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>220</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="Led" name="primGPSOmniHPLed" native="true">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>420</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="Led" name="primGPSPosMaxLed" native="true">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>80</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="Led" name="primGPSVelRejectedLed" native="true">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>140</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="Led" name="primGPSPPPLed" native="true">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>480</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_52">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>480</y>
     <width>251</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Primary GPS in PPP mode</string>
   </property>
  </widget>
  <widget class="Led" name="dmiFailedLed" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>480</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_68">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>480</y>
     <width>241</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>DMI failed or is offline</string>
   </property>
  </widget>
  <widget class="Led" name="iinWLLed" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>500</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="Led" name="iinNLLed" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>520</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_69">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>500</y>
     <width>241</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>IIN WL ambiguity error</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_75">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>520</y>
     <width>241</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>IIN NL ambiguity error</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Led</class>
   <extends>QWidget</extends>
   <header>visualization/Led.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
